tools/deployment/onnx2tensorrt.py

```diff
@@ -140,3 +140,13 @@ def parse_args():
         fp16_mode=args.fp16,
         verify=args.verify,
         workspace_size=args.workspace_size)
+
+    import warnings
+    from colorama import Fore, Back, Style
```

Comment:
The `colorama` is not a dependency of mmcls. Please use `try` to handle `ImportError`.
```suggestion
    try:
        from colorama import Fore, Style, Back
    except ImportError:
        Style = mmcv.ConfigDict({'BRIGHT': '\x1b[1m', 'RESET_ALL': '\x1b[0m'})
        Fore = mmcv.ConfigDict({'RED': '\x1b[31m', 'BLUE': '\x1b[34m'})
        Back = mmcv.ConfigDict({'LIGHTWHITE_EX': '\x1b[107m'})
```